modules/MIPS/xmleditor/index.html

```diff
@@ -1,35 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Did you do a search to make sure that index.html is not called by another program?